plutus-ledger-api/src/PlutusLedgerApi/Common/ProtocolVersions.hs

```diff
@@ -20,6 +20,15 @@ import Data.Set qualified as Set
 import GHC.Generics (Generic)
 import Prettyprinter
 
+{- Note [Adding new builtins: protocol versions]
+
+  *** ATTENTION! ***
+  New built-in functions must initially be added under `futurePV` and should
+  only be moved to an earlier MajorProtocolVersion once they have been fully
+  implemented and costed and their release under the relevant protocol version
+  has been officially approved.
```

Comment:
Sounds like this could be a test?

Reply:
How would we test that?  I'm not averse to doing it, but it's not clear to me what we could do.